nginx/nginx.conf

```diff
@@ -108,6 +108,15 @@ http {
 
         location / {
             proxy_pass http://frontend:3000/;
+            proxy_http_version 1.1;
```

Comment:
@barthalion I'm not too sure if we want these changes

Reply:
Yes, not sure if there's any harm in having it anyway, but I guess using nginx defaults would be fine - even if they might be very conservative.